### PR TITLE
Implement new ui features

### PR DESCRIPTION
**Pull request by th3james**
Actually build some of the stuff I started designing

### CODE REVIEW

File: app/views/journeys/index.html.erb [hunk 1]
```diff
@@ -1,13 +1,26 @@
 <h1>Journeys</h1>
 
-<ul class="journeys">
+<ul class="sixteen columns journeys">
```
Comment (adammulligan) [inline app/views/journeys/index.html.erb:3]: Unless I'm misunderstanding the framework, it looks like this is a 16 column grid but only 13 are used. I wonder if that will cause any issues (particularly in responsiveness) later on?

File: test/integration/journey_show_test.rb [hunk 2]
```diff
@@ -34,4 +34,34 @@ class JourneyShowTest < ActionDispatch::IntegrationTest
...
+
+    assert_response :success
+
+    page = Nokogiri::HTML(response.body)
+
+    events = page.css('.events > li')
+
+    assert_equal(
+      event_start.strftime("%H:%M:%S"),
+      events.first.css('.time').first.content.strip,
```
Comment (adammulligan) [inline test/integration/journey_show_test.rb:56]: I wonder if we might be better off using Capybara here. Capybara has a rack test driver that doesn't require a browser being open, and provides more semantic assertions: https://github.com/jnicklas/capybara#racktest

Comment (adammulligan) [inline test/integration/journey_show_test.rb:56]: Should note that I used it recently and it's very fast.

Comment (th3james) [inline test/integration/journey_show_test.rb:56]: Sounds good to me. Presumably it's a bit more expressive too

**Pull request closed without merging**